From f34ccafce0f643f8349720186f0a063339f2b244 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Wed, 31 Jul 2019 14:04:54 +0900
Subject: [PATCH] tools/sppc: fix compile error of spp_pcap

Compiling spp_pcap requires `iblz4-dev` and `liblz4-tool`, but not
including in Dockerfiles of SPP and failed to compile. This patch is
to fix the issue.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 tools/sppc/build/ubuntu/spp/Dockerfile.16.04  | 2 ++
 tools/sppc/build/ubuntu/spp/Dockerfile.18.04  | 2 ++
 tools/sppc/build/ubuntu/spp/Dockerfile.latest | 2 ++
 3 files changed, 6 insertions(+)

diff --git a/tools/sppc/build/ubuntu/spp/Dockerfile.16.04 b/tools/sppc/build/ubuntu/spp/Dockerfile.16.04
index 27311ea..a928ac4 100644
--- a/tools/sppc/build/ubuntu/spp/Dockerfile.16.04
+++ b/tools/sppc/build/ubuntu/spp/Dockerfile.16.04
@@ -32,6 +32,8 @@ RUN apt-get update && apt-get install -y \
     libarchive-dev \
     linux-headers-$(uname -r) \
     libpcap-dev \
+    liblz4-dev \
+    liblz4-tool \
     pkg-config \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
diff --git a/tools/sppc/build/ubuntu/spp/Dockerfile.18.04 b/tools/sppc/build/ubuntu/spp/Dockerfile.18.04
index afd33f9..7a7c06b 100644
--- a/tools/sppc/build/ubuntu/spp/Dockerfile.18.04
+++ b/tools/sppc/build/ubuntu/spp/Dockerfile.18.04
@@ -33,6 +33,8 @@ RUN apt-get update && apt-get install -y \
     libarchive-dev \
     linux-headers-$(uname -r) \
     libpcap-dev \
+    liblz4-dev \
+    liblz4-tool \
     pkg-config \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
diff --git a/tools/sppc/build/ubuntu/spp/Dockerfile.latest b/tools/sppc/build/ubuntu/spp/Dockerfile.latest
index 380eb18..6f87cd1 100644
--- a/tools/sppc/build/ubuntu/spp/Dockerfile.latest
+++ b/tools/sppc/build/ubuntu/spp/Dockerfile.latest
@@ -33,6 +33,8 @@ RUN apt-get update && apt-get install -y \
     libarchive-dev \
     linux-headers-$(uname -r) \
     libpcap-dev \
+    liblz4-dev \
+    liblz4-tool \
     pkg-config \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/*
-- 
2.17.1

